pkg/tfbridge/secrets.go

```diff
@@ -12,34 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package x
+package tfbridge
 
 import (
 	"context"
 
-	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
-	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/walk"
 	shim "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
 )
 
 // Ensures resource.MakeSecret is used to wrap any nested values that correspond to secret properties in the schema. A
 // property is considered secret if it is declared Sensitive in the SchemaMap in the upstream provider. Users may also
 // override a matching SchemaInfo.Secret setting to force a property to be considered secret or non-secret.
-func MarkSchemaSecrets(ctx context.Context, schemaMap shim.SchemaMap, configInfos map[string]*tfbridge.SchemaInfo,
+func MarkSchemaSecrets(ctx context.Context, schemaMap shim.SchemaMap, configInfos map[string]*SchemaInfo,
```

Comment:
This had to move back under tfbridge package due to limitations of Go complaining about recursive package dependencies.